src/VisualStudio/Core/Def/Implementation/Experimentation/KeybindingResetDetector.cs

```diff
@@ -217,37 +260,69 @@ private void ShowGoldBar()
                               action: InfoBarClose));
         }
 
-        private ReSharperStatus IsReSharperEnabled()
+        /// <summary>
+        /// Returns true if ReSharper is installed, enabled, and not suspended.  
+        /// </summary>
+        private async Task<ReSharperStatus> IsReSharperRunningAsync(ReSharperStatus lastStatus, CancellationToken cancellationToken)
```

Comment:
💡 Use `ValueTask<ReSharperStatus>` since there is a synchronous fast-path return